docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
+    {label:'List item 3', value: 'Value'},
+    {label:'List item 4', value: 'Value'},
+    {label:'List item 5', value: 'Value'},
+  ]}
+  label=''
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Order the list items in SelectList either alphabetically or by usage."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Alberto', value: 'alberto'},
+    {label:'Annie', value: 'annie'},
+    {label:'Ashley', value: 'ashley'},
+    {label:'Ayesha', value: 'ayesha'},
+    {label:'Ryan', value: 'ryan'},
+  ]}
+  label='People who are awesome'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList if there are fewer than 4 items in the list and there is space to display all options. Use RadioButtons instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'No', value: 'no'},
+    {label:'Absolutely not', value: 'noagain'},
+  ]}
+  label='Is this a good use of a SelectList?'
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the same type of selection for a group of items. An example of this might be a filter bar. If some items could use SelectList and some items need to use Dropdown, use Dropdown for all the items in the group."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='Filter 1'
+  />
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv21'},
+      {label:'Filter value 2', value: 'fv22'},
+    ]}
+    placeholder='Filter 2'
+  />
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Mix Dropdown and SelectList in a group of items."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='SelectList'
+  />
+  <Button
+    accessibilityControls="basic-dropdown-example"
+    accessibilityHaspopup
+    iconEnd="arrow-down"
+    text="Dropdown"
+    inline
+  />
+</Flex>`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      SelectList comes with Label built-in: just use the \`label\` prop. Be sure to provide a unique id so the Label is associated with the correct SelectList.`}
```

Comment:
Label isn't a component so I think I'd lower case it. 
`id` is  prop so I'd format it \`id\`

Reply:
Love it. Just made that change.

Reply:
Label is a component: https://gestalt.netlify.app/Label

And it's what we're using internally here

Reply:
Oh, interesting. OK, so maybe revert back?

Reply:
Actually, did some digging, we use `FormLabel` in this component - which I'm assuming is different from `Label`?

Reply:
FormLabel is a wrapper around Label, so this is still using Label under the hood (it wouldn't make sense to mention FormLabel, since that's a purely internal component)

Reply:
OK, switching back. Thanks!

Reply:
Looks great, though agreed with Alberto that we also need backticks around `id` to call out that it's a prop

Reply:
Ah, I missed piece of feedback - sorry and thanks!